CI.rst

```diff
@@ -901,7 +901,7 @@ Scheduled build flow
 Adding new Python versions to CI
 --------------------------------
 
-In 2.0 line we currently support Python 3.6, 3.7, 3.8.
+In 2.0 line we currently support python 3.6, 3.7, 3.8.
```

Comment:
I don't get this?

Reply:
@XD-DENG 

If i say Airflow is powered by SQLAlchemy. It is SQLAlchemy. But if i saw we must update SQLAlchemy 2.3.5, it becomes sqlalchemy 2.3.5

In the line above i get it that we are referring to Python as a tool/library rather than a product.

See issue #14283

Reply:
My question is mainly about the “Python”. To my understanding, no matter you are referring to a “tool/library” or a “product”, should be “Python” rather than “python”, unless you are talking about a command in CLI.

Reply:
@XD-DENG Wait for @mik-laj

Reply:
I agree with @XD-DENG . In this play we should use "Python", but we have "python" command line program.

Reply:
@mik-laj Why is there an exception for Python?

Reply:
Commandline are surrounded by backsticks

Reply:
here we are not talking about the command line interpreter `python`

![image](https://user-images.githubusercontent.com/8811558/109727798-e7e3f600-7bac-11eb-9432-4f88b9f99b14.png)

https://hub.docker.com/_/python

so it has to be `Python`

Reply:
I considered Python 2 and Python 3 to be products. The issue was with Python 3.x.x etc. The [official repo](https://github.com/python/cpython) says

> Documentation for Python 3.10 is online, updated daily.

Will fix